src/data/content/design-system/componenti/card.yaml

```diff
@@ -83,7 +83,22 @@ tabs:
             title: Come usarlo
             headingLevel: 2
             text: |
-              _In stesura_
+              
```

Comment:
Dobbiamo continuare a scrivere questi contenuti come elenchi lista, vedi `button`, `accordion` e altri. Non possiamo cambiare formato a +60% dell'opera @zetareticoli @danielaiozzo 
![immagine](https://github.com/italia/designers.italia.it/assets/6736103/39dd477c-b726-4315-a73c-b130a3c70bb1)